recipes/MADL-CPU-IntelMPI/README.md

```diff
@@ -0,0 +1,59 @@
+# MADL-CPU-OpenMPI
+This recipe shows how to run Microsoft Azure Distributed Linear Learner (MADL) on CPUs across
+Azure VMs via Open MPI.
+
+## Configuration
+Please see refer to this [set of sample configuration files](./config) for
+this recipe.
+
+### Pool Configuration
+The pool configuration should enable the following properties:
+* `vm_size` should be a CPU-only instance, 'STANDARD_D2_V2'.
+* `inter_node_communication_enabled` must be set to `true`
+* `max_tasks_per_node` must be set to 1 or omitted
+* `publisher` should be `Canonical` 
```

Comment:
You can omit publisher/offer/sku since there is no requirement to use a specific host.

Reply:
I also have thoroughly tested this deployment on non ubuntu OS.  While it might works nicely on all hosts, why take the chance.  My preference would be to leave the publisher and the offer, and change the SKU from 'sku should be 7.3' to 'sku could be 7.1 or 7.3'.  Will that work?